docker/build/cicd.ubuntu.Dockerfile

```diff
@@ -15,9 +15,9 @@ ENV GOROOT=/usr/local/go \
 RUN mkdir -p $GOPATH/src/github.com/algorand
 COPY . $GOPATH/src/github.com/algorand/go-algorand
 ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH \
-    GOPROXY=https://pkg.go.dev
+    GOPROXY=https://proxy.golang.org,https://pkg.go.dev,https://goproxy.io,direct
 WORKDIR $GOPATH/src/github.com/algorand/go-algorand
-RUN make ci-deps && make clean
```

Comment:
```
./scripts/buildtools/install_buildtools.sh
```
I think we'll need to run this for the image to work, unless you've already seen that it works without it

Reply:
Yes, this works without it for the build. (Historically it didn't, I know.)